### PR TITLE
added Bivariate Analysis

### PR DESCRIPTION
**Pull request by Dsantra92**
The notebook uploaded contains Bi-variate analysis of the features with the feature variables.

### CODE REVIEW

**Comment by Nibba2018** [comment]
Quick question @Dsantra92 , is this PR showing as an eligible contribution for hacktoberfest?

**Comment by Dsantra92** [comment]
@Nibba2018 it seems yes.
image

File: submissions/PythonScripts/bivariate_analysis.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  plt.pie(df1.area,labels = df1.index, colors = cs)
+  plt.title("Relative amount of Area destroyed per "+feature, fontsize = 20)
+  plt.show()
+
+get_pie_plot('month')
+
+"""We Observed the following from the pie chart :
+
+*   We observe that the most of damage from the forest fires occours during September and August.
+*   Apart from these months, majior forest fires occoured during July and March.
```
Comment (Nibba2018) [inline submissions/PythonScripts/bivariate_analysis.py]: Typos: `occur`, `major`

File: submissions/PythonScripts/bivariate_analysis.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+_,ax = plt.subplots(figsize = (20,8))
+sns.boxplot(x = "month", y ="area", data =df, ax = ax)
+sns.stripplot(x = "month", y ="area", data =df, ax = ax, color = "0.25")
+plt.title("Distribution of area of land affected per month", fontsize = 20)
+plt.show()
+
+"""We notice the following from the box plot :
+
+* Majiority of the forest fires occoured during August and September.These include majior forest fires and also very minimal forest fire.Forest fires during this month caused most damage to the forest.
```
Comment (Nibba2018) [inline submissions/PythonScripts/bivariate_analysis.py]: typo: `Majority`

File: submissions/PythonScripts/bivariate_analysis.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+* Chances of a Forest fires in January and November are negligible.
+
+* During May and December there were few forest fires but were quite impactful.
+
+* Few forest fires were experienced March, April, June, October  experienced  but they were mostly calm.This is evident from the mean line of the box plot which in their case lies very close to zero.
```
Comment (Nibba2018) [inline submissions/PythonScripts/bivariate_analysis.py]: The comment grammar can be improved.

File: submissions/PythonScripts/bivariate_analysis.py [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+### Analysis of forest fire area with rain
+"""
+
+numerical_analysis("rain","area")
+plt.suptitle("Analysis of impact of rain on Forest fire", fontsize =20)
+plt.show()
+
+"""* As Expected most of the fire occurs when there is no rain.
+* We can spot an outlier in the first plot which shows there is a fire even when ther is rain.
```
Comment (Nibba2018) [inline submissions/PythonScripts/bivariate_analysis.py]: typo: `there`

**Review by Nibba2018 — CHANGES_REQUESTED** [review]
Hello @Dsantra92 , Thank you for creating this PR. Great job with the analysis part. I have mentioned some typos below which applies to both the `.py` file and `.ipynb` files. Once they are fixed it will be ready to be merged.

**Comment by Dsantra92** [comment]
@Nibba2018 I am sorry for those typos.I have fixed the typos you mentioned as well as some other mistakes I made, I hope it's okay now.

**Comment by Nibba2018** [comment]
@Dsantra92 It's okay typos happen, you don't have to be sorry about it :laughing:. So far LGTM. Merging!

**Pull request merged**